patches/01-ocf-mbedtls-config-x509san-anon-psk.patch

```diff
@@ -80,8 +80,8 @@ index 87b4e9192..0223dce8f 100644
 +#include "port/oc_connectivity.h"
 +#include "port/oc_assert.h"
 +
-+#define MBEDTLS_KEY_EXCHANGE_ECDH_ANON_ENABLED
 +#ifdef OC_PKI
++#define MBEDTLS_KEY_EXCHANGE_ECDH_ANON_ENABLED
```

Comment:
ANON doesn't depend on PKI

Reply:
Had to enable `MBEDTLS_ECP_C` to make it compilable.